README.md

```diff
@@ -41,6 +45,11 @@ g.V().limit(10).valueMap().toList()
 [aws/graph-notebook]: https://github.com/aws/graph-notebook
 [aws/neptune-python-utils]: https://github.com/awslabs/amazon-neptune-tools/tree/master/neptune-python-utils
 
+## Versioning
+
+We use the [semantic-versioning] for tagging the code of a
+belonging to a release. Each release in git has its corresponding tag in this [dockerhub repository].
```

Comment:
It seems that the phrase is not complete or something. "for tagging the code of a belonging to a release" does not make sense.

Reply:
Yeah something is happening here.